### ISSUE TITLE
submittion error meet by docker

### ISSUE THREAD

**Issue by Open-Debin** [ebadrian/metadl#3]
hello when I submit my code to the CodaLab use your API, I meet the below error. My code runing well in the ingestion phrase, while meet error in score phrase. Please see the error below. 

docker: Error response from daemon: OCI runtime create failed: container_linux.go:349: starting container process caused "process_linux.go:449: container init caused \"process_linux.go:432: running prestart hook 1 caused \\\"error running hook: exit status 1, stdout: , stderr: nvidia-container-cli: initialization error: nvml error: driver/library version mismatch\\\\n\\\"\"": unknown.

Also, see my Pytorch Code.

import os
import logging
import csv
import gin
import torch
import numpy as np
from networks import Classifier
from networks import Conv4 as feature_encoder
from helper import image_transform, LoadParameter
import pdb
import tensorflow as tf
import torch.nn.functional as F

from metadl.api.api import MetaLearner, Learner, Predictor
@gin.configurable
class MyMetaLearner(MetaLearner):

    def __init__(self, embedding_dim=64):
        super().__init__()
        self.embedding_fn = feature_encoder(num_classes=embedding_dim)
        self.classifier = Classifier(fea_dim=embedding_dim).to(torch.device("cuda"))
        self.embedding_fn = self.embedding_fn.to(torch.device("cuda"))
        
    def meta_fit(self, meta_dataset_generator) -> Learner:
        return MyLearner(embedding_fn=self.embedding_fn, classifier=self.classifier)

@gin.configurable
class MyLearner(Learner):

    def __init__(self, embedding_dim=64, embedding_fn=None, classifier=None):
        super().__init__()
        if embedding_fn == None:
            self.embedding_fn = feature_encoder(num_classes=embedding_dim)
            self.embedding_fn = self.embedding_fn.cuda()
        else:
            self.embedding_fn = embedding_fn
            
        if classifier == None: 
            self.classifier  = Classifier(fea_dim=embedding_dim).to(torch.device("cuda"))
            self.classifier = self.classifier.cuda()
        else:
            self.classifier = classifier

    def save(self, model_dir):
        if(os.path.isdir(model_dir) != True):
            raise ValueError(('The model directory provided is invalid. Please'
                + ' check that its path is valid.'))
        
        clf_file = os.path.join(model_dir, 'learner_clf.ckpt')
        embedding_file = os.path.join(model_dir, 'learner_encoder.ckpt')
        torch.save(self.classifier.state_dict(), clf_file)
        torch.save(self.embedding_fn.state_dict(), embedding_file)
            
    def load(self, model_dir):
        if(os.path.isdir(model_dir) != True):
            raise ValueError(('The model directory provided is invalid. Please'
                + ' check that its path is valid.'))
        self.classifier.load_state_dict(torch.load(os.path.join(model_dir, 'learner_clf.ckpt')))
        self.embedding_fn.load_state_dict(torch.load(os.path.join(model_dir, 'learner_encoder.ckpt')))
        
        self.classifier = self.classifier.to(torch.device("cuda"))
        self.embedding_fn = self.embedding_fn.to(torch.device("cuda"))
        
    def fit(self, dataset_train) -> Predictor:
        for image, label in dataset_train:
            pass
        image = self.tf2torch_tensor(image).to('cuda', non_blocking=True)
        label = torch.tensor(np.array(label)).to('cuda', non_blocking=True)
        spt_fea = self.embedding_fn(image, True)[1]
        self.classifier.fit(spt_fea, label)
        return MyPredictor(self.embedding_fn, self.classifier)
    
    def tf2torch_tensor(self, image):
        image = torch.tensor(np.array(image)).permute(0,3,1,2).contiguous()
        return image
    
@gin.configurable   
class MyPredictor(Predictor):

    def __init__(self, embedding_fn, classifier):
        super().__init__()
        self.embedding_fn = embedding_fn
        self.classifier = classifier
        
    def predict(self, dataset_test):
        for image in dataset_test:
            image = self.tf2torch_tensor(image[0]).to('cuda', non_blocking=True)
            qry_fea = self.embedding_fn(image, True)[1]
            probs = self.classifier.predict(qry_fea).cpu().detach().numpy()
            probs = tf.convert_to_tensor(probs)
        return probs
    
    def tf2torch_tensor(self, image):
        image = torch.tensor(np.array(image)).permute(0,3,1,2).contiguous()
        return image
